docs/layouts/shortcodes/generated/dynamic_section.html

```diff
@@ -182,5 +182,11 @@
             <td>Map</td>
             <td>Custom HTTP header for HttpArtifactFetcher. The header will be applied when getting the session job artifacts. Expected format: headerKey1:headerValue1,headerKey2:headerValue2.</td>
         </tr>
+        <tr>
+            <td><h5>kubernetes.operator.plugins.listeners.<listener-name>.class</h5></td>
```

Comment:
Actually, these html files are generated instead of changing manually.

As I understand, you need to update the `KubernetesOperatorConfigOptions` and run `mvn clean install -DskipTests -Pgenerate-docs`.

Reply:
Thank you for your advice. It seems I have some misconceptions about its construction. I will fix it。